RIGS/migrations/0035_auto_20191008_2148.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.13 on 2019-10-08 20:48
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Why does this file exist in the PR? There is already a similar migration in `master`

Reply:
makemigrations on the asset branch makes it for whatever reason, investigating.

Reply:
https://github.com/nottinghamtec/PyRIGS/commit/1a49bb50e5c1a0625d67f33e0ce7464820441f3f contains a change to a model, after which we branched off to assets. Meant that when `makemigrations` was next run, it caused it to be created on next run.

No worries, will have to make sure it is removed after merge!